bin/rrd2whisper.py

```diff
@@ -32,21 +32,21 @@
 option_parser = optparse.OptionParser(usage='''%prog rrd_path''')
 option_parser.add_option(
     '--xFilesFactor',
-    help="The xFilesFactor to use in the output file. " +
-    "Defaults to the input RRD's xFilesFactor",
+    help="The xFilesFactor to use in the output file. \
+          Defaults to the input RRD's xFilesFactor",
     default=None,
     type='float')
 option_parser.add_option(
     '--aggregationMethod',
-    help="The consolidation function to fetch from on input and " +
-    "aggregationMethod to set on output. One of: %s" %
+    help="The consolidation function to fetch from on input and \
+          aggregationMethod to set on output. One of: %s" %
     ', '.join(aggregationMethods),
     default='average',
     type='string')
 option_parser.add_option(
     '--destinationPath',
-    help="Path to place created whisper file. Defaults to the " +
-    "RRD file's source path.",
+    help="Path to place created whisper file. Defaults to the \
+          RRD file's source path.",
```

Comment:
Don't these changes effectively add a chunk of whitespace in the middle of the strings?

Reply:
Argparse and optparse reformat the help text so it doesn't matter. I must admit that the new flake error confuses me, I'm not sure how they expect you to format it.

Initially it gave:
> ./bin/rrd2whisper.py:41:5: W504 line break after binary operator                         

So I first moved the operator to the second line but then got: 
> ./bin/rrd2whisper.py:42:11: W503 line break before binary operator

Reply:
Yeah I just disabled W504 in carbon and graphite-web